pytest_rerunfailures.py

```diff
@@ -175,6 +175,19 @@ def get_reruns_delay(item):
     return delay
 
 
+def get_reruns_condition(item):
+    rerun_marker = _get_marker(item)
+
+    condition = True
+    if rerun_marker is not None and "condition" in rerun_marker.kwargs:
+        condition = rerun_marker.kwargs["condition"]
+
+    if not isinstance(condition, bool):
+        warnings.warn("Condition has to be a boolean expression.")
```

Comment:
What is your intend with this warning? Above in the README you propose, that all values evaluating to `True` should be recognized as valid condition.

Reply:
Right, good point! I'm excluding a bunch of values that will evaluate to `True` or `False` here.

Maybe we should leave out the check (and warning) and just rely on the documentation?

Reply:
I am fine with the documentation .

Reply:
Removed and changed the documentation.